climada/entity/exposures/litpop/litpop.py

```diff
@@ -0,0 +1,1108 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Define LitPop class.
+"""
+import logging
+import numpy as np
+import rasterio
+import geopandas
+from shapefile import Shape
+from shapely.geometry import Polygon, MultiPolygon
+from pathlib import Path
+import pandas as pd
+
+import climada.util.coordinates as u_coord
+from climada.util.finance import gdp, income_group, wealth2gdp, world_bank_wealth_account
+from climada.entity.tag import Tag
+from climada.entity.exposures.litpop import nightlight as nl_util
+from climada.entity.exposures.litpop import gpw_population as pop_util
+from climada.entity.exposures.base import Exposures, INDICATOR_IMPF
+from climada.util.constants import SYSTEM_DIR
+LOGGER = logging.getLogger(__name__)
+
+class LitPop(Exposures): # TODO tests nighlight (integ), gpw_population (integ) and litpop (integ)
+    """Defines exposure values from nightlight intensity (NASA), Gridded Population
+        data (SEDAC); distributing produced capital (World Bank), population count,
+        GDP (World Bank), or non-financial wealth (Global Wealth Databook by Credit Suisse
+        Research Institute.)
+
+        Calling sequence example:
+        ent = LitPop()
+        country_names = ['CHE', 'Austria']
+        ent.set_countries(country_names)
+        ent.plot()
```

Comment:
It is an Exposures right, not an entity?

Also, could you please add the list of attributes.